src/components/structures/VectorEmbeddedPage.js

```diff
@@ -29,6 +29,7 @@ export default class VectorEmbeddedPage extends EmbeddedPage {
     translate(s) {
         s = sanitizeHtml(_t(s));
         // ugly fix for https://github.com/vector-im/riot-web/issues/4243
+        // eslint-disable-next-line max-len
         s = s.replace(/\[matrix\]/, '<a href="https://matrix.org" target="_blank" rel="noreferrer noopener"><img width="79" height="34" alt="[matrix]" style="padding-left: 1px;vertical-align: middle" src="welcome/images/matrix.svg"/></a>');
```

Comment:
why does this even exist 😭 

(not an issue for this PR, just expressing my sadness for such a regex)

Reply:
I have no idea